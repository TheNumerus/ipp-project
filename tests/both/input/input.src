.ippcode20
DEFVAR GF@a
DEFVAR GF@b
WRITE string@Zadej\032A:\032
READ GF@a int
WRITE string@\010Zadej\032B:\032
READ GF@b int
DEFVAR GF@c
ADD GF@c GF@a GF@b
WRITE string@\010
WRITE GF@a
WRITE string@\032+\032
WRITE GF@b
WRITE string@\032=\032
WRITE GF@c
WRITE string@\010