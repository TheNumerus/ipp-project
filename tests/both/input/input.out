Zadej A: 
Zadej B: 
5 + 6 = 11
